Lib/test/test_urllib2.py

```diff
@@ -211,6 +213,25 @@ def test_password_manager(self):
         self.assertEqual(find_user_pass("Some Realm", "e.example.com:3128"),
                          ('5', 'e'))
 
+        # is_suburi
+
+        true = self.assertTrue
+        false = self.assertFalse
+        true(is_suburi(reduce_uri('http://example.com'),
```

Comment:
It would be easier to understand the logic tested if there was a blank line between each subtest.